docs/0.3/grid_purchase_order.md

```diff
@@ -1,158 +1,194 @@
-
-- `grid_id` is the purchase order's UUID in Grid.
-
-## Purchase Order Versions and Revisions
-
-The `PurchaseOrderVersion` represents a particular version of a purchase order
-as it is iterated upon. A `PurchaseOrderVersion` may have multiple
-`PurchaseOrderRevision`s associated with it, with one being the current
-revision. The `PurchaseOrderRevision` contains the business data for the
-purchase order. Any update to the business data of a purchase order will result
-in the creation of a new revision of a version.
-
-The permissions around when and how new `PurchaseOrderVersion`s and
-`PurchaseOrderRevision`s are created will depend on which purchase order
-workflow is being used by the purchase order.
-
-### PurchaseOrderVersion representation in Grid
-
-```protobuf
-message PurchaseOrderVersion {
-  string version_id = 1;
-  string workflow_status = 2;
-  bool is_draft = 3;
-  string current_revision_id = 4;
-  PurchaseOrderRevision revisions = 5;
-}
-```
-
-- `version_id` is the globally unique ID of a purchase order version.
-
-- `workflow_status` is the status of the purchase order version within the
-  workflow. The possible values of the status will depend on which workflow is
-  in use.
-
-- `is_draft` determines whether the purchase order version is still being
-  iterated on. The workflow may determine what actions are possible to take on
-  a purchase order based on this status.
-
-- `current_revision_id` is the identifier of the most recent revision.
-
-- `revisions` is a list of all of the revisions that have been created for a
-  version.
-
-### PurchaseOrderRevision representation in Grid
-
-```protobuf
-message PurchaseOrderRevision {
-  string revision_id = 1;
-  string submitter = 2;
-  uint64 created_at = 3;
-
-  string order_xml_v3_4 = 4;
-}
-```
-
-- `revision_id` is the unique identifier for the revision.
-- `submitter` is the public key of the agent who submitted this revision.
-- `created_at` acts as a timestamp of when the revision was created. This has
-  the same stipulations as the `created_at` property of `PurchaseOrder`, as
-  described above.
-- `order_xml_v3_4` holds a string consisting of the GS1 3.4 purchase order XML
-  data. This is the core business data of the system. Note that this data is
-  opaque to the purchase order smart contract. However, the Grid CLI parses and
-  validates this data against the proper XSD. Other clients should be sure to
-  do the same parsing and validation.
+# Grid Purchase Order
+
+<!--
+  Copyright 2022 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+Grid Purchase Order is a smart contract that enables buyers and sellers to
+automate and synchronize their purchase order processes and data. It aims
+to reduce administrative overhead, improve transaction accuracy and speed, and
+reduce the number of disputes, ultimately improving business relationships.
+
+### Contents
```

Comment:
This section should be removed, it duplicates the right sidebar.

Reply:
Thanks, fixed